person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(params)
     @name = validate_name params[:name] || 'Unknown'
     @age = params[:age]
     @parent_permission = params.fetch(:parent_permission, true)
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
- [ ] In order to properly show the `has-many` relationship between person and rental, getters and setters for rentals should be present.